test/e2e/fixtures/gencert.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Sorry, I don't understand where this is used? Is it to generate the certs above after they expire? I think the same thing for the `gencert.json`, `ca.csr`, and `ca-csr.json` above.

Also, should we consider using `testdata` for the directory rather than fixtures?

> Directory and file names that begin with "." or "_" are ignored by the go tool, as are directories named "testdata".

- from https://golang.org/cmd/go/#hdr-Package_lists_and_patterns

Reply:
@shawn-hurley it is used to generate CA key and Cert which is then used by my tests.

```go
func init() {
	caCertBytes, err := ioutil.ReadFile("./fixtures/ca.crt")
	if err != nil {
		panic(err)
	}
	caConfigMap.Data = map[string]string{tlsutil.TLSCACertKey: string(caCertBytes)}
 	caKeyBytes, err := ioutil.ReadFile("./fixtures/ca.key")
	if err != nil {
		panic(err)
	}
	caSecret.Data = map[string][]byte{tlsutil.TLSPrivateCAKeyKey: caKeyBytes}
}
```

`should we consider using testdata for the directory rather than fixtures?`
sounds good.

Reply:
fixed! PTAL cc/ @shawn-hurley